docs/testing-guidelines/WritingPesterTests.md

```diff
@@ -82,7 +82,7 @@ However, if you need to check the `InnerException` or other members of the Error
 For creation of PowerShell tests, the Describe block is the level of granularity suggested and one of three tags should be used: "CI", "Feature", or "Scenario". If the tag is not provided, tests in that describe block will be run any time tests are executed.
 
 #### Describe
-Creates a logical group of tests. All Mocks and TestDrive contents defined within a Describe block are scoped to that Describe; they will no longer be present when the Describe block exits. A `Describe block may contain any number of Context and It blocks.
+Creates a logical group of tests. All Mocks and TestDrive contents defined within a Describe block are scoped to that Describe; they will no longer be present when the Describe block exits. A `Describe` block may contain any number of Context and It blocks.
```

Comment:
Please add a newline after the `#### Describe` header to make the Markdown valid to all parsers

Reply:
the `markdownlint` extension will find these issue in VS Code